### PR TITLE
start of RFC document

### PR DESCRIPTION
**Pull request by AndrewAnnex**
## Description
as per https://github.com/USGS-Astrogeology/ISIS_TC/pull/146, this PR begins the addition of a RFC policy document to be expanded upon. 

## Motivation and Context
We need a document to explain the RFC process better. 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [ ] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

**Review by jessemapel — DISMISSED** [review]

File: RFC.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# RFC
+The ISIS project uses a Request for Comment (RFC) model whereby major potential changes to the code base, data area, or binary delivery process are proposed, iterated on by any interested parties, and potentially adopted. Right now, RFCs are being housed n this repository's [discussion board](https://github.com/USGS-Astrogeology/ISIS3/discussions?discussions_q=label%3ARFC).
```
Comment (jlaura) [inline RFC.md]: s/housed n this/housed in this

File: RFC.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+We encourage all contributors and users to review open RFCs and comment as these proposed changes will impact use of the software.
```
Comment (jlaura) [inline RFC.md]: comment,

File: RFC.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+We encourage all contributors and users to review open RFCs and comment as these proposed changes will impact use of the software. 
+
+# RFC policies
+
+RFC adoption is driven by general consensus in the RFC comments.
+
+# RFC lifecycle
+
+1. RFC is written and posted on the discussion board
+2. RFC is advertised and discussion proceeds until consensus is reached on what changes if any are needed
```
Comment (jlaura) [inline RFC.md]: RFC is advertised and discussion proceeds until consensus is reached (the rest of the sentence does not make sense)

File: RFC.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+3. RFC is closed, subsiquent issues/pull requests reference RFC. 
+
+```markdown
+- Feature/Process Name: (Name of this RFC)
+- Start Date: (Today's Date)
+- RFC PR: (empty until a PR is opened)
+- Author: (your name)
+
+# Summary
+Include a brief summary of the RFC so readers can determine if the RFC is of interest.
```
Comment (jlaura) [inline RFC.md]: Make the prompts comments so the user can just copy/paste?

**Review by jlaura — DISMISSED** [review]

**Review by jessemapel — APPROVED** [review]

**Pull request merged**